rollup.utils.js

```diff
@@ -184,33 +184,69 @@ function magicExportsPlugin({ packageName, version }) {
       }
 
       let banner = createBanner(packageName, version);
+      let moduleName = camelCase(packageName.slice("@remix-run/".length));
       let esmContents = banner + "\n";
       let tsContents = banner + "\n";
       let cjsContents =
         banner +
         "\n" +
         "'use strict';\n" +
-        "Object.defineProperty(exports, '__esModule', { value: true });\n";
+        "Object.defineProperty(exports, '__esModule', { value: true });\n\n";
 
       if (magicExports.values) {
-        let exportList = magicExports.values.join(", ");
-        esmContents += `export { ${exportList} } from '${packageName}';\n`;
-        tsContents += `export { ${exportList} } from '${packageName}';\n`;
+        let deprecationFunctions =
+          // eslint-disable-next-line no-template-curly-in-string
+          "const getDeprecatedMessage = (symbol, packageName) => `All \\`remix\\` exports are considered deprecated as of v1.3.3. Please import \\`${symbol}\\` from \\`${packageName}\\` instead. You can run \\`remix migrate --migration replace-remix-imports\\` to automatically migrate your code.`;\n" +
```

Comment:
We'll merge this before the next minor release.
```suggestion
          "const getDeprecatedMessage = (symbol, packageName) => `All \\`remix\\` exports are considered deprecated as of v1.8.0. Please import \\`${symbol}\\` from \\`${packageName}\\` instead. You can run \\`remix migrate --migration replace-remix-imports\\` to automatically migrate your code.`;\n" +
```

Reply:
Since we use `v1.3.3` in our ESLint warnings as well, I kept it here too

https://github.com/remix-run/remix/blob/983836112ac7400e53cb6f35b2cbf37035d716f2/packages/remix-eslint-config/rules/core.js#L16-L17